app/src/main/java/edu/uc/kansakpb/ikeahjalpare3/service/FurnitureService.kt

```diff
@@ -12,7 +12,7 @@ class FurnitureService {
 
     internal fun fetchFurnitures(furnitureName: String) : MutableLiveData<ArrayList<Furniture>> {
         //MVD is defined
-        var _furnitures = MutableLiveData<ArrayList<Furniture>>()
+        val furnitures = MutableLiveData<ArrayList<Furniture>>()
```

Comment:
Variable name shouldn't start with underscore.

Reply:
This is OK, if it is wrapped by a public variable.  But, since this is a variable in the scope of a function, it won't be wrapped by a public variable... so I agree, remove the _